samples/java/src/main/java/com/example/Main.java

```diff
@@ -0,0 +1,80 @@
+package com.example;
+
+import com.example.account.BankAccount;
+import com.example.account.CreateAccount;
+import com.example.account.CreditAccount;
+import com.example.account.DebitAccount;
+import com.example.command.BankAccountCommand;
+import com.example.event.BankAccountEvent;
+import surge.javadsl.command.SurgeCommand$;
+import surge.javadsl.command.SurgeCommand;
+import surge.javadsl.common.CommandFailure;
+import surge.javadsl.common.CommandResult;
+import surge.javadsl.common.CommandSuccess;
+
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ExecutionException;
+
+public class Main {
+    private static BankAccount BankAccount;
+
+    public static void main(String[] args) throws ExecutionException, InterruptedException {
+        BankAccountSurgeModel bankAccountSurgeModel = new BankAccountSurgeModel();
+        SurgeCommand<UUID, BankAccount, BankAccountCommand, ?, BankAccountEvent> surgeCommand =
+                SurgeCommand$.MODULE$.create(bankAccountSurgeModel);
+        surgeCommand.start();
+
+        UUID accountNumber = UUID.randomUUID();
+        System.out.println(accountNumber);
```

Comment:
would it be possible to replace the System.out.println statements with log info statements ?

Reply:
added log statement